### PR TITLE
apps wc: falco-alert routing to alertmanager in sc

### PR DESCRIPTION
**Pull request by robinelastisys**
**What this PR does / why we need it**:
Falco alerts get routed through Prometheus to the alert manager in the wc-cluster as it cannot be delivered by falcon itself.

**Which issue this PR fixes** 
fixes elastisys/ck8s-ops#106

**Special notes for reviewer**:
This could be developed further. Instead of having one generic FalcoAlert-rule, we could have one alert for each rule defined in  https://github.com/falcosecurity/charts/blob/d7f1fbb31b94ecc28cfd1693bdcf8d867ce6d39d/falco/rules/falco_rules.yaml. Is it wanted?

**Checklist:**

- [x] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [x] Proper commit message prefix on all commits

### CODE REVIEW

**Review by lentzi90 — COMMENTED** [review]
Great job!
I don't think there is a point in making one alert rule per rule. It is simple and effective like this! :slightly_smiling_face: 

Note that the short syntax for linking to the issue (fixes #...) only works within the same repo, so you need to add the full URL to make it work in this case. :slightly_frowning_face:  I.e. fixes https://github.com/elastisys/ck8s-ops/issues/106

**Comment by robinelastisys** [comment]
The suggested changes have been made! :+1: :slightly_smiling_face:

File: helmfile/charts/prometheus-alerts/templates/rules/falco-alerts.yaml
Comment (lentzi90) [reply helmfile/charts/prometheus-alerts/templates/rules/falco-alerts.yaml]: Sorry for not making this clear the first time. You need to add this new value [to the values](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/charts/prometheus-alerts/values.yaml#L15-L43). So just like you see `kubernetesResources: true` there, we need `falcoAlerts: true`. This is just to set the default value, otherwise it would be undefined and helm would complain about it.

Comment (robinelastisys) [reply helmfile/charts/prometheus-alerts/templates/rules/falco-alerts.yaml]: Done! :+1:

**Review by lentzi90 — APPROVED** [review]
Great!

Please squash the commits, then this is good to go for me. :slightly_smiling_face: 
Maybe best to wait for one more review to get a second opinion also before merging.

File: helmfile/charts/prometheus-alerts/templates/rules/falco-alerts.yaml
Comment (OlleLarsson) [inline helmfile/charts/prometheus-alerts/templates/rules/falco-alerts.yaml:3]: You could get rid of this if you want.

**Review by OlleLarsson — COMMENTED** [review]
LGTM.

**Review by OlleLarsson — APPROVED** [review]

**Pull request merged**
